OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -21,12 +21,12 @@ namespace OpenRA.Mods.Common.Activities
 {
 	public class Move : Activity
 	{
+		public WAngle ActorFacingModifier;
```

Comment:
Does this need to be public? And if so it should at least be `{ get; private set; }`.

Reply:
Fixed